x-pack/plugins/infra/public/services/telemetry/telemetry_events.ts

```diff
@@ -118,21 +118,55 @@ const assetDetailsFlyoutViewed: InfraTelemetryEvent = {
     componentName: {
       type: 'keyword',
       _meta: {
-        description: 'Hostname for the clicked host.',
+        description: 'Asset name for the clicked asset.',
```

Comment:
Nice that you fixed it. How about this?
```suggestion
        description: 'Name of the parent react component for the clicked asset.',
```